src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.cs

```diff
@@ -687,6 +692,7 @@ internal void FinishOperationSyncSuccess(int bytesTransferred, SocketFlags flags
                 LogBuffer(bytesTransferred);
             }
```

Comment:
This is good example of why the split across event sources is problematic.  We're tracking bytes transferred with one event source but logging the contents with another.